### PR TITLE
Create dave9123.json

### PR DESCRIPTION
**Pull request by dave9123**

### CODE REVIEW

File: domains/dave9123.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+{
+  "owner": {
+    "username": "dave9123",
+    "discord": "Dave9123#0757",
```
Comment (MaskDuck) [inline domains/dave9123.json:5]: i would like to see an email here.

if you dont have or dont wanna leak, fill in the email with an empty string like this: `{"email": ""}`

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/dave9123.json (hunk 1 above)
Comment (dave9123) [reply domains/dave9123.json:5]: Where should I place?

**Review by dave9123 — COMMENTED** [review]
Better?

**Review by MaskDuck — APPROVED** [review]
not sus

**Review by mtgsquad — APPROVED** [review]
Seems good to me.

**Comment by dave9123** [comment]
Thanks

**Comment by dave9123** [comment]
Why
image

**Comment by MaskDuck** [comment]
netlify doesnt work with is-a-dev

**Comment by dave9123** [comment]
Or make it to dave9123.tk?

**Comment by MaskDuck** [comment]
maybe you can try [railway](https://railway.app)

**Comment by dave9123** [comment]
I used another freenom domain and dave9123.tk works

**Comment by dave9123** [comment]
But Idk

**Comment by dave9123** [comment]
No idea on how to use railway

**Pull request merged**